packages/Input/src/Input.styles.js

```diff
@@ -21,122 +21,122 @@ const iconStyles = {
   `,
 };
 
-const inputStyles = css`
-  line-height: 1;
-  position: relative;
-
-  input.form-input__input {
-    ${stylers.placeholders}
-
-    background-color: ${tokens.color.white};
-    border: 1px solid ${tokens.border.color};
-    border-radius: ${tokens.border.radius};
-    box-shadow: none;
-    box-sizing: border-box;
-    color: ${tokens.color.black};
-    display: block;
-    margin: 0;
-    padding: 0 0 0 ${tokens.space};
-    padding-right: ${({ hasClearButton }) => (hasClearButton ? stylers.spacer(3) : tokens.space)};
-    transition: box-shadow 0.2s, color 0.2s;
-    width: 100%;
-
-    &:focus {
-      background-color: ${tokens.color.white};
-      border-color: ${tokens.highlight.active.noBorder.borderColor};
-      box-shadow: ${tokens.highlight.active.noBorder.boxShadow};
-      outline: none;
-    }
-
-    &::-ms-clear {
-      display: none;
-    }
-  }
+export const InputWrapper = styled.div`
+  ${props => `
+    line-height: 1;
+    position: relative;
 
-  /* Sizes */
+    input.form-input__input {
```

Comment:
You don't need to do it in this PR, it's a bit out of scope, but thought I'd call it out that this "className" approach is a legacy left over from migrating the component from `acl-ui`, and one we'd like to avoid going forward by simply exporting another styled component for this element and applying it in the component.